setup.py

```diff
@@ -136,6 +136,7 @@ def qd_config(arg):
         'astropy>=5.0.4',
         'numpy',
```

Comment:
To be consistent with doc.
```suggestion
        'numpy>=1.18',
```